app/src/main/kotlin/hr/asee/android/template/compose/ui/postlogin/users/Giver.kt

```diff
@@ -0,0 +1,65 @@
+package hr.asee.android.template.compose.ui.postlogin.users
+
+import hr.asee.android.template.compose.ui.common.service.Offer
+import hr.asee.android.template.compose.ui.common.service.ParkingSpace
+import hr.asee.android.template.compose.ui.common.service.Reservation
+import hr.asee.android.template.compose.ui.common.service.Seeking
+import java.time.LocalDateTime
+
+class Giver(
+    id: Int = generateId(),
+    firstName: String,
+    lastName: String,
+    email: String,
+    password: String,
+    activated: Boolean = true,
+    langKey: String = "English",
+    createdBy: String = "admin",
+    createdDate: LocalDateTime,
+    lastModifiedBy: String = createdBy,
+    lastModifiedDate: LocalDateTime = createdDate,
+    authorities: List<String>? = null,
+    resetDate: LocalDateTime? = null
+):
+    User(
+        id = id,
+        firstName = firstName,
+        lastName = lastName,
+        email = email,
+        password = password,
+        activated = activated,
+        langKey = langKey,
+        createdBy = createdBy,
+        createdDate = createdDate,
+        lastModifiedBy = lastModifiedBy,
+        lastModifiedDate = lastModifiedDate,
+        authorities = authorities,
+        resetDate = resetDate
+    ) {
+
+    var reservations = mutableSetOf<Reservation>(
+    )
+    var offers = mutableSetOf<Offer>(
+    )
+    var requests = mutableSetOf<Seeking>(
+    )
+    var parkingSpaces = mutableSetOf<ParkingSpace>(
+    )
```

Comment:
User ne bi trebao imati ove varijable, ako si to stavio privremeno samo za testiranje, onda okej, ali inace neće ovog biti